core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/LibraryDesign.java

```diff
@@ -31,13 +29,11 @@ public static boolean validate(Library library, Iterable<LibraryDesign> rules) {
   private Long librarySelectionType;
   @Column(nullable = false)
   private Long libraryStrategyType;
+  @OneToOne(targetEntity = LibraryType.class)
+  @JoinColumn(name = "libraryType", nullable = false)
+  private LibraryType libraryType;
   @Column(nullable = false)
   private String name;
-  @Column(nullable = false)
-  private Boolean paired;
-  @Enumerated(EnumType.STRING)
-  @Column(nullable = false)
-  private PlatformType platformName;
   @OneToOne(targetEntity = SampleClassImpl.class)
```

Comment:
Since `paired` does need to be set on the object, can we use LibraryType to infer that? Need to talk to the lab...